client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -152,10 +158,9 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 
 	const categorizationOptions = getCategorizationOptions(
 		intent,
-		showDesignPickerCategoriesAllFilter,
-		showGeneratedDesigns
+		showDesignPickerCategoriesAllFilter || showGeneratedDesigns
```

Comment:
I think we don't need to fall back to `showGeneratedDesigns`?

Reply:
I've refactored that part a bit, so we don't need the `showGeneratedDesign` in this line,